gcloud-java-storage/src/main/java/com/google/gcloud/storage/testing/RemoteGcsHelper.java

```diff
@@ -96,14 +96,48 @@ public static String generateBucketName() {
   }
 
   /**
+   * Creates a {@code RemoteGcsHelper} object for the given project id and JSON key input stream.
+   *
+   * @param projectId id of the project to be used for running the tests
+   * @param keyStream input stream for a JSON key
+   * @return A {@code RemoteGcsHelper} object for the provided options.
+   * @throws com.google.gcloud.storage.testing.RemoteGcsHelper.GcsHelperException if
+   *     {@code keyStream} is not a valid JSON key stream
+   */
+  public static RemoteGcsHelper create(String projectId, InputStream keyStream)
```

Comment:
Strange, looks like my original comment was lost.
Here it is again:
Do you think that there is still value in providing Options which now only includes a flag to read from the classpath?
I think it should be simple enough for someone to just pass `getClass().getResourceAsStream(keyPath)` directly to this method...